javascript/operators/optional_chaining.json

```diff
@@ -8,10 +8,24 @@
           "spec_url": "https://tc39.es/proposal-optional-chaining/#top",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "78",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Experimental JavaScript",
```

Comment:
```suggestion
                  "name": "Enable Experimental JavaScript Features",
```

Reply:
Sure, but i can't figure out where you get that name from? When i go to `chrome://flags`, the header for the flag is just called `Experimental JavaScript`, and the id is `#enable-javascript-harmony`.

Reply:
I see the same thing as Anton. What versions of Chrome are you two using? They've been known to change flag names.

Reply:
I'm seeing the same thing in the current versions of Chrome 77, 78, 79, 80

Reply:
Ah, my bad!  I was basing the flag name on what we’ve had in BCD already — remind me to double-check Chrome flags before writing reviews... 😅